packages/crayons-core/src/components/data-table/custom-cells/user/custom-cell-user.tsx

```diff
@@ -0,0 +1,32 @@
+import { Component, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'fw-custom-cell-user',
+  styleUrl: 'custom-cell-user.scss',
+  shadow: true,
+})
+export class CustomCellUser {
```

Comment:
CustomCellUser class name is already present in the class above. Why have two classes with same name? Is that even allowed? Does this compile?

Reply:
This was a mistake from my side. Changed 'CustomCellAnchor'. This compiled without issues, that is why I didn't notice the mistake last time.